CONTRIBUTING.md

```diff
@@ -98,3 +110,26 @@ The website listens on port 8081.
 
 The best place to get help is to come [join the discord](https://discord.gg/FHpfstT7Dw). There is
 a "contributors" channel dedicated to helping contributors.
+
+## Common errors
+
+We've seen a few errors in the past that you may encounter:
+
+### Error: Cannot find module '/path/to/tybalt/packages/cli/dist/index.js'
+
+This happens when you try to run a command like `serve` or `test` that requires the cli to have been compiled. Run `yarn run build` to compile the project.
+
+### /bin/sh: line 1: npm: command not found
+
+Although we use yarn, one of our dependencies, esbuild, assumes that npm is available at install time on your path. Make sure to add npm to your path or workspace.
+
+### Error: EEXIST: file already exists, mkdir 'path/to/tybalt/node_modules/aria-build/node_modules/esbuild/.install'
+
+This can happen if the Nix process flakes. Remove all untracked files and reinstall the dependencies:
```

Comment:
@doug-wade we can remove this particular error for now, as I am not really sure about why this particular error was triggered. And since not everyone uses Nix, this is a distro-specific issue. I'll make a simple flake.nix for other Nix users out there.